velox/exec/tests/JoinFuzzerRunner.h

```diff
@@ -56,12 +58,28 @@
 class JoinFuzzerRunner {
  public:
   static int run(size_t seed) {
-    facebook::velox::memory::MemoryManager::initialize({});
+    setupMemory();
     facebook::velox::serializer::presto::PrestoVectorSerde::
         registerVectorSerde();
     facebook::velox::filesystems::registerLocalFileSystem();
 
     facebook::velox::exec::test::joinFuzzer(seed);
     return RUN_ALL_TESTS();
   }
+
+ private:
+  // Invoked to set up memory system with arbitration.
+  static void setupMemory() {
```

Comment:
Could we use `ArbitratorTestUtil::createMemoryManager` here?

Reply:
@xiaoxmeng Would you answer this question?